samples/sync_query.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function  # For python 2/3 interoperability
+from samples.utils import get_service, query_paging, poll_query
+
+
+# [START sync_query]
+def sync_query(service, project_id, query, timeout=0):
+    """ Run a synchronous query"""
+    query_data = {
+                    'query': query,
+                    'timeoutMs': timeout,
+                 }
+
+    return service.jobs().query(
+            projectId=project_id,
+            body=query_data).execute()
```

Comment:
Does this make polling synchronous queries totally unnecessary? I was under the impression that the method blocks, and returns as soon as the query is complete (or fails) in which case the only point of polling is to retry automatically. If this is the case, I'd love to completely get rid of the polling queries topic.